lib/routers/establishment/projects.js

```diff
@@ -179,6 +188,12 @@ router.post('/:id/fork',
   submit('fork')
 );
 
+router.put('/:id/update-licence-holder',
+  perms('project.update'),
+  whitelist('licenceHolderId'),
```

Comment:
Don't we also want all the other fields?

Reply:
no, they are meta innit

Reply:
Ahhh... I see what you did there.